qa/no-bootstrap-tests/src/test/java/org/elasticsearch/bootstrap/SpawnerNoBootstrapTests.java

```diff
@@ -70,7 +72,7 @@ static class ExpectedStreamMessage implements MockLogAppender.LoggingExpectation
         final String expectedLogger;
         final String expectedMessage;
         final CountDownLatch matchCalledLatch;
```

Comment:
nit: the member name implies it should be decremented when the match method is called. Perhaps it should not be something liked "matchedLatch"?